src/renderer/reducers/register.ts

```diff
@@ -1,21 +1,37 @@
-import {IRegisterAction} from "../actions";
+import {ISigningKeyMnemonicAction, ISigningKeyAction, IWithdarawalKeyAction} from "../actions";
 import {RegisterActionTypes} from "../constants/action-types";
+import {Action} from "redux";
 
 export interface IRegisterState {
-    mnemonic: string
+    mnemonic: string,
+    signingKey: string,
+    withdrawalKey: string
 }
 
-const initalState: IRegisterState = {
-    mnemonic: ""
+const initialState: IRegisterState = {
+    mnemonic: "",
```

Comment:
Do we need this, you can always convert immediately to signingKey?

Reply:
Hmmm, @irubido  did this in the first place, I also was wondering if we gonna need it, but then I realized that he will maybe use it in mnemonic verify? Ivan can you confirm this

Reply:
@jogamod yes, we need this for mnemonic verifying. I'll open an issue to pass mnemonic by url param

Reply:
yes, I used it to pass mnemonic